src/content/docs/de/basics/layouts.mdx

```diff
@@ -1,41 +1,51 @@
+import ReadMore from '~/components/ReadMore.astro'
 
-Eine Layout-Komponente wird üblicherweise verwendet, um einer [`.astro`- oder `.md`-Seite](/de/basics/astro-pages/) sowohl ein **Seiten-Gerüst** (`<html>`, ` <head>` und `<body>`-Tags) als auch einen `<slot />` zur Verfügung zu stellen, der bestimmt, wo im Layout der Seiteninhalt eingefügt werden soll.
+**Layouts** sind [Astro-Komponenten](/de/basics/astro-components/), die verwendet werden, um eine wiederverwendbare UI-Struktur, wie z.B. eine Seitenvorlage, bereitzustellen.
 
-Wenn eine Layout-Komponente ein Seiten-Gerüst enthält sollte beachted werden, dass das `<html>`-Element das übergeordnete Element der Seite sein muss. Jegliche [`<style>`](/de/guides/styling/#styles-stile-in-astro)- oder [`<script>`](/de/guides/client-side-scripts/#client-seitige-skripte)-Elemente müssen vollständig vom `<html>`-Element umschlossen sein. 
+Wir verwenden den Begriff „Layout“ üblicherweise für Astro-Komponenten, die gemeinsame UI-Elemente für alle Seiten bereitstellen, wie z. B. Kopf-, Navigations- und Fußzeilen. Eine typische Astro-Layoutkomponente bietet [Astro-, Markdown- oder MDX-Seiten](/de/basics/astro-pages/) folgendes:
+- einer **Seitenhülle** (`<html>`, `<head>` und `<body>` Tags)
+- einen [**`<Slot />`**](/de/basics/astro-components/#slots), um festzulegen, wo einzelne Seiteninhalte eingefügt werden sollen.
 
-Layouts enthalten häufig gemeinsame `<head>`-Elemente und gemeinsame UI-Elemente der Website, z.B. Kopfzeilen, Navigationsleisten und Fußzeilen.
+Allerdings sind Layoutkomponenten nichts Besonderes! Sie können [Props akzeptieren](/de/basics/astro-components/#props-komponenteneigenschaften) und [andere Komponenten importieren und verwenden](/de/basics/astro-components/#komponentenstruktur) wie jede andere Astro-Komponente. Sie können [UI-Framework-Komponenten](/de/guides/framework-components/) und [client-seitige Skripte](/de/guides/client-side-scripts/) enthalten. Sie müssen nicht einmal eine vollständige Seitenoberfläche bereitstellen, sondern können stattdessen als partielle UI-Vorlagen verwendet werden.
 
-Layouts werden normalerweise im Verzeichnis `src/layouts` deines Projekts abgelegt.
+Wenn eine Layoutkomponente jedoch eine Seitenhülle enthält, muss ihr `<html>`-Element das übergeordnete Element aller anderen Elemente der Komponente sein. Alle [`<style>`](/de/guides/styling/#styles-stile-in-astro) oder [`<script>`](/de/guides/client-side-scripts/#verwendung-von-script-in-astro) Elemente müssen von den `<html>` Tags umschlossen sein.
+
+Layout-Komponenten werden normalerweise in einem `src/layouts`-Verzeichnis in deinem Projekt platziert, aber das ist keine Voraussetzung; du kannst sie überall in deinem Projekt platzieren. Du kannst Layout-Komponenten sogar neben deinen Seiten platzieren, [indem du den Layout-Namen ein `_` voranstellst](/de/guides/routing/). 
```

Comment:
```suggestion
Layout-Komponenten werden normalerweise in einem `src/layouts`-Verzeichnis in deinem Projekt platziert, aber das ist keine Voraussetzung; du kannst sie überall in deinem Projekt platzieren. Du kannst Layout-Komponenten sogar neben deinen Seiten platzieren, [indem du den Layout-Namen ein `_` voranstellst](/de/guides/routing/#excluding-pages). 
```

The link seems to be different. If there is no heading that corresponds to the link, it seems like a good idea to keep the original link.

Reply:
My thought process was:

- If there is no german file, link to the english heading.
- If there is a finished translation, link to the same locale heading.

But in this case, there is an outdated german version where this heading doesn't even exist yet... So linking to either the german or english heading will always land on top of the german page.

Is it now really better to link to the english heading, or should I already link to the german one (which I will translate in the future)?

Reply:
It's okay to keep it, it's just a suggestion.

The reason for the suggestion is that if you update the `routing.mdx` file after this PR is merged, you may forget that you need to update this file again.

If you add an English version of the heading, CI might throw an error when you update the `routing.mdx` file and realize that you need to update this file along with it.

Reply:
Okay, this is a very inspiring way of keeping such things in mind! I wrote a TODO comment below this line, but I think I'll delete the TODO comment and update it. Thank you!